temporal-serviceclient/src/main/java/io/temporal/internal/retryer/GrpcRetryerUtils.java

```diff
@@ -76,6 +76,8 @@ class GrpcRetryerUtils {
         // By default, we keep retrying with DEADLINE_EXCEEDED assuming that it's the deadline of
         // one attempt which expired, but not the whole sequence.
         break;
+      case RESOURCE_EXHAUSTED:
```

Comment:
Why do we need to add it here? Do we want to prohibit users to make RESOURCE_EXHAUSTED not retryable if they want? Why?

Reply:
Don't remember why and can't reproduce now. Removing.